src/components/Queue.vue

```diff
@@ -5,12 +5,15 @@
         <h1 v-on:click="collapsed = !collapsed" class="expandable queue" v-bind:class="{'expanded' : !collapsed}">Queued vouchers</h1>
 
         <transition name="fade" v-if="show">
-            <div v-if="!message" class="goodmessage queue">
-                You have <strong>{{ vouchers.length }}</strong> vouchers in your queue.
+            <div v-if="fail && message" class="goodmessage error">
```

Comment:
Are you sure? goodmessage and error?

@gingerbreadpup For start of next sprint - would be good to define what types of messages we can have and how to implement them. We seem to be going in circles on this every time a new person implements a message.